docs/guides/nextauth-migration.mdx

```diff
@@ -0,0 +1,362 @@
+---
+title: Migrating to Clerk from NextAuth.js
+description: Migrating from NextAuth.js
+---
+
+<TutorialHero
+  framework="next"
+  beforeYouStart={[
+    {
+      title: "Set up a Clerk application",
+      link: "https://clerk.com/docs/quickstarts/setup-clerk",
+    }
+  ]}
+>
+
+- Install `@clerk/nextjs`
+- Set environment keys
+- Wrap your Next.js app in `<ClerkProvider />`
+- Setup Sign-up and Sign-in UI
+- Protect your app
+- Read user and session data
+- User IDs as Foreign Keys
+- Create a production instance
+- Migrate users to a production instance
+- Further support
+
+</TutorialHero>
+
+This guide shows how to migrate an application using NextAuth.js to use Clerk for authentication.
+
+<Steps>
+
+### Install `@clerk/nextjs`
+
+Clerk's Next.js SDK gives you access to prebuilt [components](https://clerk.com/docs/components/overview), [hooks](https://clerk.com/docs/references/nextjs/overview#client-side-helpers), and [helpers](https://clerk.com/docs/references/nextjs/overview) for Next.js Server Components, Route Handlers and Middleware. Install it by running the following command in your terminal:
```

Comment:
```suggestion
Clerk's Next.js SDK gives you access to prebuilt [components](/docs/components/overview), [hooks](/docs/references/nextjs/overview#client-side-helpers), and [helpers](/docs/references/nextjs/overview) for Next.js Server Components, Route Handlers and Middleware. Install it by running the following command in your terminal:
```

Use relative links

Reply:
> Use relative links
Does it not work otherwise? What is the reasoning?

Reply:
A full URL creates a 'leaving the site' type link.

Reply:
Makes sense, thanks for the context.